csharp/src/Drivers/BigQuery/BigQueryStatement.cs

```diff
@@ -185,6 +186,33 @@ static IArrowReader ReadChunk(BigQueryReadClient readClient, string streamName)
                 {
                     options.AllowLargeResults = true ? keyValuePair.Value.ToLower().Equals("true") : false;
                 }
+                if(keyValuePair.Key == BigQueryParameters.LargeResultsDestinationTable)
+                {
+                    string destinationTable = keyValuePair.Value;
+
+                    if (!destinationTable.Contains("."))
+                        throw new InvalidOperationException($"{BigQueryParameters.LargeResultsDestinationTable} is invalid");
+
+                    string projectId = string.Empty;
+                    string datasetId = string.Empty;
+                    string tableId = string.Empty;
+
+                    List<string> segments = destinationTable.Split('.').Where(x => x.Length > 0).ToList();
```

Comment:
I don't think "a....b....c" should be allowed.

In principle, should this follow BigQuery identifier rules for e.g. escaping? Is there a function in the Google libraries to parse table names.

Reply:
I don't see a method to parse the table names. 

BigQuery separates the full table names by periods as well, ie,

SELECT  FROM `pq-test-398321.AdbcTests.adbc_all_types` LIMIT 1000

Reply:
Sorry, I didn't do a great job explaining this feedback. What I meant was that this test would allow me to pass a..b..c (that is a dot dot be dot dot c) and have it be treated the same way as a.b.c (a dot b dot c) and that seems wrong.

Reply:
ok, fixed